src/domain/locking/locking.repository.interface.ts

```diff
@@ -20,6 +21,12 @@ export interface ILockingRepository {
     offset?: number;
   }): Promise<Page<Rank>>;
 
+  getLeaderBoardByCampaignId(args: {
```

Comment:
Nit: "leaderboard" is one word:

```suggestion
  getLeaderboardByCampaignId(args: {
```

Reply:
I'm resolving this one as this got renamed to `getCampaignLeaderboard`. Lmk if you have other suggestion of course!